modules/virt-building-real-time-container-disk-image.adoc

```diff
@@ -3,163 +3,92 @@
 // * virt//support/monitoring/virt-running-cluster-checkups.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="virt-building-real-time-container-disk-image_{context}"]
-= Building a container disk image for {op-system-base} virtual machines
+[id="virt-building-realtime-container-disk-image_{context}"]
+= Building a realtime container disk image for {op-system-base} virtual machines
 
-You can build a custom {op-system-base-full} 8 OS image in `qcow2` format and use it to create a container disk image. You can store the container disk image in a registry that is accessible from your cluster and specify the image location in the `spec.param.vmUnderTestContainerDiskImage` attribute of the real-time checkup config map.
-
-To build a container disk image, you must create an image builder virtual machine (VM). The _image builder VM_ is a {op-system-base} 8 VM that can be used to build custom {op-system-base} images.
-
-.Prerequisites
-* The image builder VM must run {op-system-base} 8.7 and must have a minimum of 2 CPU cores, 4 GiB RAM, and 20 GB of free space in the `/var` directory.
-* You have installed the image builder tool and its CLI (`composer-cli`) on the VM.
-
-* You have installed the `virt-customize` tool by using the following command:
-+
-[source,terminal]
-----
-# dnf install libguestfs-tools
-----
-* You have installed the Podman CLI tool (`podman`).
+You can create a custom {op-system-base-full} 8 Operating system image in `QCOW2` format and use it to create a container disk image. You can store the container disk image in a registry that is accessible from your cluster and specify the image location in the `spec.param.vmUnderTestContainerDiskImage` attribute of the realtime checkup config map.
```

Comment:
What is the reason to change the wording from the original text?
It is the same as in the DPDK checkup.

Reply:
it's not the same.
On the DPDK you have instruction on the VM where the image is created on, here we do not.